source/features/more-dropdown-links.css

```diff
@@ -2,8 +2,3 @@
 .rgh-has-more-dropdown .UnderlineNav-actions {
 	visibility: visible !important;
 }
-
-/* Only show the divider if there are other items above it */
-.UnderlineNav-actions [hidden] + .dropdown-divider {
-	display: none;
-}
```

Comment:
This hid the divider when `releases-tab` was hidden _but_ Security is always there anyway. I don't care to support those who disable `more-dropdown-links` for this minor inconvenience.